packages/rest-crud/src/crud-rest.controller.ts

```diff
@@ -208,7 +208,7 @@ export function defineCrudRestController<
     async create(
       @body(modelCtor, {
         title: `New${modelName}`,
-        exclude: modelCtor.getIdProperties() as (keyof T)[],
+        exclude: modelCtor.getExcludeProperties() as (keyof T)[],
```

Comment:
Do we really want to exclude `readonly` property from create methods ?

Reply:
See my [comment.](https://github.com/loopbackio/loopback-next/pull/9339#discussion_r1257276006)